src/main/java/com/rev_connect_api/controllers/PostController.java

```diff
@@ -6,21 +6,23 @@
 import com.rev_connect_api.models.Post;
 import com.rev_connect_api.services.MediaService;
 import com.rev_connect_api.services.PostService;
+
 import jakarta.validation.Valid;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
 
+import java.math.BigInteger;
 import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
```

Comment:
nitpick: don't change the import statements unless we are actually removing or adding imports. if the ide does this automatically, we probably want to find the same settings across ides for auto-importing